src/services/crons/aws.query.cache.warmer.service.ts

```diff
@@ -121,7 +121,6 @@ export class AWSQueryCacheWarmerService {
             this.logger.info('Start refresh pairs analytics');
             const profiler = new PerformanceProfiler();
             for (const pairAddress of pairsAddresses) {
-                this.logger.info(`Refresh analytics for pair: ${pairAddress}`);
```

Comment:
isn`t this log helpful?

Reply:
Not really. It was only for debugging purposes